sql/_07_misc/_01_statistics/answers/1031.answer

```diff
@@ -13,11 +13,13 @@ db_user
 null     
 
 ===================================================
-0
+Error:-158
```

Comment:
you may change sql cases,to avoid error code,such as

call add_user('test_user') on class db_user;

create class t1(c1 int);

create class t2(c1 int);

create class t3(c1 int);
grant alter on t1,t2,t3 to test_user; 

call login('test_user') on class db_user;

update statistics on t1, t2, t3;

call login('dba') on class db_user;

drop class t1, t2, t3;

call drop_user('test_user') on class db_user;